GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeCoordinator.swift

```diff
@@ -54,13 +54,14 @@ extension CallVisualizer {
                     case .closeButtonTap:
                         self?.codeViewController?.presentingViewController?.dismiss(animated: true)
                         self?.delegate?(.closeTap)
+                    case .refreshButtonTap:
```

Comment:
Why this decision is made in coordinator? I would suggest to move it inside view model, since it is the one responsible for to business logic.

Reply:
fixed [9c8a7dd](https://github.com/salemove/ios-sdk-widgets/pull/459/commits/9c8a7dd8d4bd3e407e572ffcc5e61e33b4868ccc)